syntax  = "proto3";

package fixbug;

option cc_generic_services = true;

// 配置序列化反序列化protol;

message Result
{
    int32 errcode = 1;
    bytes errmessage = 2;
}

message LoginRequest
{
    bytes name = 1;
    bytes pwd = 2;
}

message LoginResponse
{
    Result res = 1;
    bool success = 2;
}

// 新建一对register类 供userServiceRpc来调用
message RegisterRequest{
    int32 id = 1;
    bytes name = 2;
    bytes pwd = 3;
}

message RegisterRepose{
    Result res = 1;
    bool success = 2;
}


service UserServiceRpc
{
    rpc Login(LoginRequest) returns(LoginResponse);
    rpc Register(RegisterRequest) returns(RegisterRepose);
}